### ISSUE TITLE
colliding filepaths

### ISSUE THREAD

**Issue by aerorahul** [NOAA-EMC/GSI#104]
The following files have the same name and pose a problem on a case insensitive filesystem e.g. macOS.

Please consider renaming one or the other with a distinct name.

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'util/Conventional_Monitor/image_gen/ush/Transfer.sh'
  'util/Conventional_Monitor/image_gen/ush/transfer.sh'
```

**Comment by EdwardSafford-NOAA** [comment]
Fair enough.  I'll rename the lower case file name.

**Comment by EdwardSafford-NOAA** [comment]
I've run this using the GFS/gdas ConMon image files and the transfer operation works fine.

**Comment by MichaelLueken-NOAA** [comment]
Closing issue since the update has been merged to the master at 76aa328.
